<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_sys_manage_outer_pay_set_sql</class>
 <widget class="QDialog" name="w_sys_manage_outer_pay_set_sql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_2"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <item>
      <widget class="lds_ComboBox" name="comboBox_sql_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_sql_run">
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
